pyproject.toml

```diff
@@ -10,7 +10,7 @@ authors = ["Your Name <you@example.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "3.10"
```

Comment:
Ce changement cause l'erreur : 
`Current Python version (3.10.10) is not allowed by the project (3.10).`